CodeEdit/Features/About/Views/AboutView.swift

```diff
@@ -27,31 +35,144 @@ public struct AboutView: View {
 
     private static var licenseURL = URL(string: "https://github.com/CodeEditApp/CodeEdit/blob/main/LICENSE.md")!
 
+    func getScrollAdjustedValue(
+        minValue: CGFloat,
+        maxValue: CGFloat,
+        minOffset: CGFloat,
+        maxOffset: CGFloat
+    ) -> CGFloat {
+        let valueRange = maxValue - minValue
+        let offsetRange = maxOffset - minOffset
+        let currentOffset = scrollOffset
+        let percentage = (currentOffset - minOffset) / offsetRange
+        let value = minValue + (valueRange * percentage)
+
+//        TODO: Commented out for now to move with overscroll effect, should this be configurable functionality?
+//        if currentOffset >= minOffset {
+//            return minValue
+//        }
+
+        if currentOffset <= maxOffset {
+            return maxValue
+        }
+        if value < 0 {
+            return 0
+        }
+        return value
+    }
+
+    let smallTitlebarHeight: CGFloat = 28
+    let mediumTitlebarHeight: CGFloat = 113
+    let largeTitlebarHeight: CGFloat = 231
+
+    let maxScrollOffset: CGFloat
+
+    public init() {
+        self.maxScrollOffset = -self.mediumTitlebarHeight + self.smallTitlebarHeight
+    }
+
     public var body: some View {
```

Comment:
Maybe split the `body` into different section (private var/func) where it makes sense to make it easy to distinguish between them and keep things organized.

Reply:
I have a body length violation so I definitely planned to do that.